calculater.js

```diff
@@ -0,0 +1,8 @@
+let w  = 10;
+let h = 20;
+let q = 30;
+let p = 40;
+console.log (w+h); //Addition/
```

Comment:
great! You'll want to put a space after your group of variable declarations, and also space around the operators.

```let w  = 10;
let h = 20;
let q = 30;
let p = 40;

console.log (w+h); ```